invenio_records_rest/views.py

```diff
@@ -88,6 +89,19 @@ def elasticsearch_badrequest_error(error):
                 return handler(error)
         return error
 
+    for exception_cls, handlers in (error_handlers_registry or {}).items():
```

Comment:
Maybe fix the default for the argument at the top of the function? ``error_handlers_registry = error_handlers_registry or {}``